autokeras/nn/model_trainer.py

```diff
@@ -33,6 +36,7 @@ def __init__(self,
                  loss_function,
                  train_data,
                  test_data=None,
+                 timeout=0,
```

Comment:
0 here may not be safe. Can we change it to none?

Reply:
Sure, I'll use none to represent the no timeout